internal/datastore/postgres/postgres.go

```diff
@@ -375,6 +378,98 @@ func (pgd *pgDatastore) ReadWriteTx(
 	return datastore.NoRevision, err
 }
 
+const repairTransactionIDsOperation = "transaction-ids"
+
+func (pgd *pgDatastore) Repair(ctx context.Context, operationName string, outputProgress bool) error {
+	switch operationName {
+	case repairTransactionIDsOperation:
+		return pgd.repairTransactionIDs(ctx, outputProgress)
+
+	default:
+		return fmt.Errorf("unknown operation")
+	}
+}
+
+const batchSize = 10000
```

Comment:
do we know the impact of such batch size on a production postgres? should this be at a more conservative value and expose a flag?